src/styles.css

```diff
@@ -52,16 +52,23 @@ h4 {
   margin-bottom: 0;
 }
 
-.addToList {
-  width: 90%;
-  padding-left: 8px;
+.task-container {
```

Comment:
- [ ] [OPTIONAL] I found out that the CSS styles are not working. Please try to fix it when possible. Have a great day 👍 and welcome back 👏.